panther_description/urdf/wheel.urdf.xacro

```diff
@@ -31,15 +38,15 @@
 
     <joint name="${prefix}_wheel_joint" type="continuous">
```

Comment:
The joints of the wheels cannot have `$ns` before their name due to the  configuration of the `diff_drive_controller` in `WH0x_controller.yaml` file. Am I right?

Reply:
I add to this [PR](https://github.com/ros-controls/ros2_controllers/pull/1104) similar logic, where ros2_controll automaticlly add tx_prefix or namespace before joint name.